pint/models/spindown.py

```diff
@@ -25,29 +25,28 @@ def __init__(self):
         # The number of terms in the taylor exapansion of spin freq (F0...FN)
         self.num_spin_terms = maxderivs
 
```

Comment:
It's actually not a big deal, though.  There are similar limits in TEMPO (and probably in T2).  If you have more derivs than that, you should't be using a Taylor expansion...